index.html

```diff
@@ -28,6 +27,132 @@ <h1>Hemerson Foreste</h1>
+              </div>
+            </div>
+  
+            <div class="project-division">
+              <img src="images/Laptop-Right.png" alt="laptops">
+              <div class="project-name">
+                <h4>Project name goes here</h4>
+                <ul class="language-info">
+                  <li class="html-css">HTML/CSS</li>
+                  <li class="ruby">Ruby on Rails</li>
+                  <li class="javascript">JavaScript</li>
+               </ul>
+               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
+              </div>
+            </div>
+  
+            <div class="project-division">
+              <img src="images/Laptop-Left.png" alt="laptops">
+              <div class="project-name">
+                <h4>Project name goes here</h4>
+                <ul class="language-info">
+                  <li class="html-css">HTML/CSS</li>
+                  <li class="ruby">Ruby on Rails</li>
+                  <li class="javascript">JavaScript</li>
+               </ul>
+               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
+              </div>
+            </div>
+  
+            <div class="project-division">
+              <img src="images/Laptop-Pink.png" alt="laptops">
+              <div class="project-name">
+                <h4>Project name goes here</h4>
+                <ul class="language-info">
+                  <li class="html-css">HTML/CSS</li>
+                  <li class="ruby">Ruby on Rails</li>
+                  <li class="javascript">JavaScript</li>
+               </ul>
+               <button type="button"><span>See this project</span><span><img class="arrow-right" src="images/Union.svg" alt="arrow right"></span></button>
+              </div>
+            </div>          
+        </section>
+  
+        <!--About Section-->
+        <section id="about">
+            <h2>About</h2>
+            <p class="about-content">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+              labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+            </p>
+            <button type="button"><span>Get my CV</span><span><img class="arrow-download" src="images/ic_download.svg" alt="arrow download"></span></button>
+            <div id="Languages">
+                <h3>Languages</h3>
+              <ul>
+                  <li>JavaScript(ES5/ES6)</li>
+                  <li>Ruby</li>
+                  <li>HTML</li>
+                  <li>CSS</li>
+              </ul>
+              <img id="img-languages" src="images/diese.png" alt="#">
+            </div>
```

Comment:
- [ ] The content of the languages should aligned with flexbox. **_You should apply it to frameworks and skills_** [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=In%20order%20to%20lay%20out%20the%20elements%20in%20%22Works%22%20section%20you%20should%20use%20CSS%20Grid%20and%20Flexbox%20combined%20(it%20is%20mandatory%20that%20you%20use%20both).%20Use%20CSS%20Grid%20in%20the%20main%20container%20that%20contains%20all%20of%20the%20cards%20and%20use%20Flexbox%20to%20lay%20out%20an%20individual%20card.)